GluuFederation/docker-gg
# 4.1.0_dev

## What does this pr do?  
Updates to the new Gluu-gateway 4.1

## How should this be manually tested

1. Create a common docker network 
 `docker network create kong-net`

 2. Start your database
 `docker run -d --name kong-database --network kong-net -p 5432:5432 -e "POSTGRES_USER=kong" -e "POSTGRES_DB=kong" postgres:9.6`

3. Prepare your database.
    `docker run --rm \
     --network=kong-net \
     -e "KONG_DATABASE=postgres" \
     -e "KONG_PG_HOST=kong-database" \
     -e "KONG_CASSANDRA_CONTACT_POINTS=kong-database" \
     gluufederation/gluu-gateway:4.1.0_dev kong migrations bootstrap`

4. Start gg-api
     ` docker run -d --name kong \
     --network=kong-net \
     -e "KONG_DATABASE=postgres" \
     -e "KONG_PG_HOST=kong-database" \
     -e "KONG_CASSANDRA_CONTACT_POINTS=kong-database" \
     -e "KONG_PROXY_ACCESS_LOG=/dev/stdout" \
     -e "KONG_ADMIN_ACCESS_LOG=/dev/stdout" \
     -e "KONG_PROXY_ERROR_LOG=/dev/stderr" \
     -e "KONG_ADMIN_ERROR_LOG=/dev/stderr" \
     -e "KONG_ADMIN_LISTEN=0.0.0.0:8001, 0.0.0.0:8444 ssl" \
     -p 8000:8000 \
     -p 8443:8443 \
     -p 8001:8001 \
     -p 8444:8444 \
     gluufederation/gluu-gateway:4.1.0_dev`

4. Test if gg is up, running and ready to accept connections
    `curl -i http://localhost:8001/` 
     The expected output for this command if GG is running correctly is shown in the screenshot below.

## Screenshots if available

<img width="1435" alt="Screenshot 2019-12-31 at 17 52 45" src="https://user-images.githubusercontent.com/13331207/71624997-75693600-2bf6-11ea-9ffc-051e7c012936.png">
